src/components/TrackedCourses/TrackedCoursesView.tsx

```diff
@@ -37,30 +37,49 @@ class TrackedCoursesView extends React.Component {
     }
   }
 
+  updateSections = async (courseId: number) => {
+    const updatedAvailableSections = this.state.availableSections.filter(section => section.catalogNum != courseId)
+    const updatedAwaitingSections = this.state.awaitingSections.filter(section => section.catalogNum != courseId)
+    this.setState({
+      availableSections: updatedAvailableSections,
+      awaitingSections: updatedAwaitingSections
+    })
+  }
+
   render() {
     const availableSectionsView = this.state.availableSections.map(section => {
       return (
-        <TrackedCourseCard section={section} />
+        <TrackedCourseCard
+          key={section.catalogNum}
+          section={section}
+          untrackSectionHandler={(courseId) => this.updateSections(courseId)}
+        />
       )
     })
     const awaitingSections = this.state.awaitingSections.map(section => {
       return (
-        <TrackedCourseCard section={section} />
+        <TrackedCourseCard
+          key={section.catalogNum}
+          section={section}
+          untrackSectionHandler={(courseId) => this.updateSections(courseId)}
```

Comment:
I think remove parens around single params like you do on lines 41/2.

```suggestion
          untrackSectionHandler={courseId => this.updateSections(courseId)}
```